scripts/sysman/ras.yml

```diff
@@ -40,6 +40,15 @@ etors:
       desc: "The number of errors that have occurred in caches (L1/L3/register file/shared local memory/sampler)"
     - name: DISPLAY_ERRORS
       desc: "The number of errors that have occurred in the display"
+    - name: MEMORY_ERRORS
+      desc: "The number of errors that have occurred in Memory"
+      version: "1.7"
+    - name: SCALE_ERRORS
```

Comment:
Please share if we could rename this to SCALEFABRIC_ERRORS

Reply:
might be confusing since we have "L3FABRIC_ERRORS" below.updated desc to remove "Fabric" wording